src/rpc/chain.ts

```diff
@@ -997,11 +997,13 @@ export class ChainRpc {
      * Gets pending transactions that have the insertion timestamp within the given range.
      * @param from The lower bound of the insertion timestamp.
      * @param to The upper bound of the insertion timestamp.
+     * @param futureIncluded is designed
```

Comment:
Would you make it better? FYI, these comments will be shown in the https://api.codechain.io

https://api.codechain.io/2.0.1/classes/chainrpc.html#getpendingtransactions

Reply:
Do you want me to change the name of the parameter?

Reply:
I meant the description. The format is `* @param ${paramName} ${description}`

Currently, the description is "is designed" and it's not a sentence.